### ISSUE TITLE
Add alias analysis information to parallel diagnostics reporting.

### ISSUE THREAD

**Issue by stuartarchibald** [numba/numba#3774]
## Feature request

As title.

CC: @anton-malakhov @DrTodd13

**Comment by anton-malakhov** [comment]
yep! the other ideas were to put this report into annotated html along with adding/reusing the command line argument of `numba` entry point. Duplicating all the rest of env var options as `numba` command line arguments would be perfectionist's dream ;-)

**Comment by stuartarchibald** [comment]
@anton-malakhov please could you open a new ticket for the env vars on the `numba` command line tool?
